google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -131,7 +129,8 @@
   private final UnaryCallSettings<Query, Row> readRowSettings;
   private final UnaryCallSettings<String, List<KeyOffset>> sampleRowKeysSettings;
   private final UnaryCallSettings<RowMutation, Void> mutateRowSettings;
-  private final BatchingCallSettings<RowMutation, Void> bulkMutateRowsSettings;
+  private final BatchingCallSettings<RowMutationEntry, Void, BulkMutation, Void>
+      bulkMutateRowsSettings;
```

Comment:
As discussed offline, this is pending investigation. We need to future proof this for adding latency based throttling. Before we submit this PR, we should have a plan for configuring the throttle latency

Reply:
I am on it, Once we finalize on approach for this we can merge this.